client/js/lounge.js

```diff
@@ -1,3 +1,15 @@
+import $ from "jquery";
+import "jquery-ui/ui/widgets/sortable";
+import Mousetrap from "mousetrap";
+import Handlebars from "handlebars/runtime";
+import "notification-polyfill";
+
+import "./libs/jquery/inputhistory";
+import "./libs/jquery/tabcomplete";
+import "./libs/jquery/stickyscroll";
```

Comment:
I believe all these libs by @erming are on npm, but also that we have monkey-patched these over time. A great contribution would be diff-ing ours with the upstream ones (because, yeah, they have evolved on the upstream project too...), open PRs on @erming's repos, and as they are being merged, move these to use the npm versions :-)